packages/flutter_tools/test/general.shard/migrations/swift_package_manager_integration_migration_test.dart

```diff
@@ -3011,8 +3304,50 @@ String migratedNativeTargetSection(
     '			productReference = 97C146EE1CF9000F007C117D /* Runner.app */;',
     '			productType = "com.apple.product-type.application";',
     '		};',
-    '/* End PBXNativeTarget section */',
   ].join('\n');
+
+  builder.writeln('/* Begin PBXNativeTarget section */');
+
+  builder.writeln(runnerTarget);
+
+  if (otherNativeTarget != null) {
+    builder.writeln(otherNativeTarget);
+  }
+
+  builder.write('/* End PBXNativeTarget section */');
+
+  return builder.toString();
+}
+
+/// A variant of [migratedNativeTargetSection]
+/// that includes an additional PBXNativeTarget, "OtherTarget", that is not migrated.
+String migratedRunnerWithUnmigratedOtherTargetSection(SupportedPlatform platform) {
+  final String otherTarget = <String>[
```

Comment:
I only added this target for an iOS specific test. Not sure if we need a MacOS variant also? The only real difference is the identifiers being slightly different.